webapp-django/crashstats/signature/views.py

```diff
@@ -424,3 +424,112 @@ def signature_graph_data(request, channel):
     )
 
     return data
+
+
+@waffle_switch('signature-report')
+def signature_summary(request):
+    '''Return a list of specific aggregations. '''
+    params = get_validated_params(request)
+    if isinstance(params, http.HttpResponseBadRequest):
+        # There was an error in the form, let's return it.
+        return params
```

Comment:
This function is used in a lot of place. I agree that it's not Pythonic, and I don't remember why but I know I had a "good" reason to do it like this. I don't know if doing this refactoring in a PR that's already 450 lines long is a good idea though.